src/pages/AlumniPage.jsx

```diff
@@ -1,90 +1,179 @@
+import alum_navbar from "../assets/alum_navbar.png";
+import alum_panel from "../images/alum_panel.png";
+import alum_scholarship from "../assets/alum_scholarship.png";
+import alum_picnic from "../images/alum_picnic.png";
+import alum_picnic2 from "../assets/alum_picnic2.png";
+import fep_award from "../images/fep_award.png";
+import alum_triangle from "../assets/alum_triangle.png";
+import alum_dots1 from "../assets/alum_dots1.png";
+import alum_dots2 from "../assets/alum_dots2.png";
+import alum_dots3 from "../assets/alum_dots3.png";
+import alum_vector1 from "../assets/alum_vector1.png";
+import alum_vector2 from "../assets/alum_vector2.png";
+import alum_circles1 from "../assets/alum_circles1.png";
+import alum_circles2 from "../assets/alum_circles2.png";
+import alum_circles3 from "../assets/alum_circles3.png";
+import alum_circles4 from "../assets/alum_circles4.png";
+import alum_shadow1 from "../assets/alum_shadow1.png";
+import alum_shadow2 from "../assets/alum_shadow2.png";
+import alum_shadow3 from "../assets/alum_shadow3.png";
+import alum_rectangle from "../assets/alum_rectangle.png";
+import rod from "../images/rod.png";
+import alum_facebook from "../assets/alum_facebook.png";
+import alum_linkedin from "../assets/alum_linkedin.png";
+import alum_email from "../assets/alum_email.png";
+
+
+
+
 
 export function AlumniPage() {
     return (
-        <div className="flex flex-col items-center justify-center w-full min-h-screen bg-white p-8 overflow-x-hidden">
-            <div className="max-w-[1088px] w-full flex flex-col justify-center items-center gap-10">
-                <h1 className="text-black text-center font-inter text-4xl font-medium mt-20" style={{ letterSpacing: '8px' }}>
-                    ALUMNI NETWORK
-                </h1>
-                <p className="text-black text-center font-inter text-base leading-7 px-32">
-                    The Networking Directors of FUSION host a variety of programs to connect the alumni network to FUSION’s current active members, 
-                    to give our members the opportunity to connect and learn from our alumni from everything to professional development to general life advice.
-                </p>
+        <div className="flex flex-col overflow-x-hidden">
+            <Navbar title="ALUMNI" subtitle="PROGRAMS" bg={alum_navbar} />
+            {/* Give Back and Get Involved */}
+            <div className="w-full h-[540px] bg-white flex justify-center items-center">
+                <div className="w-[58%] h-1/2 text-center">
+                    <div className="font-semibold text-5xl">
+                        <span className="text-[#FFA523]">Give Back</span> <span className="text-black">and Get Involved</span>
+                    </div>
+                    <hr className="w-48 h-1 mx-auto bg-[#FAC80A] border-0 rounded my-10"/>
+                    <div className="h-1/2 flex justify-center items-center">
+                        <p className="text-black text-center">
```

Comment:
The vertical spacing needs to be a little bit more big, theres something with line alignment on tailwindcss documentation that can help you with this